src/peergos/server/SpaceCheckingKeyFilter.java

```diff
@@ -504,6 +504,26 @@ private static void processRemovedOwnedKeys(State state,
         }
     }
 
+    @Override
+    public CompletableFuture<Long> getUsage(PublicKeyHash owner) {
+        Stat stat = state.currentView.get(owner);
+        if (stat == null)
+            return CompletableFuture.completedFuture(0L);
+        Usage usage = state.usage.get(stat.owner);
+        if (usage == null)
+            return CompletableFuture.completedFuture(0L);
```

Comment:
It would be clearer to throw an error in this case to disambiguate unknown-user a user that has taken up 0 bytes of space.

Reply:
A user cannot use 0 space. Just signing up uses ~28k.

Reply:
But an error is probably clearer, will change.